website/docs/handle_exceptions.md

```diff
@@ -0,0 +1,51 @@
+---
+id: handle_exceptions
+title: handle_exceptions Decorator
+sidebar_label: handle_exceptions Decorator
+---
+
+## handle_exceptions Decorator
+`Core-Lib`'s `web_helpers` provides `handle_exception` decorator, that logs the exception, and it's error message and returns a `Http Response Object` 
+with error message in case of any exception. 
+
+```python
+def handle_exceptions(func):
+```
+This decorator can be used when for handling exceptions when making requests to the db or server to log the exception and 

```

Comment:
`requests to the db or server to log the exception and` what if i connect to a telnet server. can i use this? 

we are already explaining what this decorator is 3 lines above